src/Main/Settings.cpp

```diff
@@ -24,10 +24,24 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include <QFile>
 #include <iostream>
 
+#include "NewProject/ProjectManager/config.h"
+
 using namespace FOEDAG;
 
+#define SETTINGS_DEBUG false
```

Comment:
you can use qDebug() << ""; Looks weird.

Reply:
These are rare use debug prints that are only needed when actively developing in this area so i don't want the prints coming out everytime someone is running debug.